src/handlers/importProductsFile/handler.ts

```diff
@@ -0,0 +1,33 @@
+import { PutObjectCommand, S3Client } from '@aws-sdk/client-s3';
+import { APIGatewayProxyEvent } from 'aws-lambda';
+import { BUCKET, TYPE_CSV } from '../../constants';
+import { generateHttpResponse } from '../../utils/lambda';
+import { StatusCode } from '../../types/http';
+import { getSignedUrl } from "@aws-sdk/s3-request-presigner";
+
+const s3 = new S3Client();
+
+export const importProductsFile = async (event: APIGatewayProxyEvent) => {
+  const fileName = event.queryStringParameters ? event.queryStringParameters.name : null;
```

Comment:
You can just write

`const fileName = event.queryStringParameters?.name`

Reply:
Done